go/tools/builders/generate_test_main.go

```diff
@@ -214,6 +218,7 @@ func genTestMain(args []string) error {
 	goenv := envFlags(flags)
 	out := flags.String("output", "", "output file to write. Defaults to stdout.")
 	coverage := flags.Bool("coverage", false, "whether coverage is supported")
```

Comment:
Can you remove the flag `coverage` and use `"cover_mode" != ""` to decide whether coverage is enabled?

Reply:
Yes, of course. It's done.